packages/api/src/db/page.ts

```diff
@@ -98,7 +104,7 @@ export interface GetPageHistoryArgs {
   readonly id: string
 }
 
-export interface PageInput extends PageData {}
+export type PageInput = PageData
```

Comment:
@bishoymelek is this your editor??? You might need to change some of your automatic editor behavior.

Reply:
that's the warning i get:
```
An interface declaring no members is equivalent to its supertype.eslint@typescript-eslint/no-empty-interface
```